tests/e2e/resources/kserve/inference-service.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: serving.kserve.io/v1beta1
+kind: InferenceService
+metadata:
+  annotations:
+    sidecar.istio.io/inject: 'false'
+  name: half-plus-two
+  namespace: kubeflow-example-com
+spec:
+  predictor:
+    model:
+      modelFormat:
+        name: tensorflow
+      storageUri: s3://s3-kserve-xugq6/saved_model_half_plus_two/
```

Comment:
This shouldn't be hardcoded right? Artifact of testing?

Reply:
yes you are right. it got set up in here: https://github.com/awslabs/kubeflow-manifests/blob/299b3a8c874f213668f8ea1fe1553f9eed2fca50/tests/e2e/fixtures/kserve_denpendencies.py#L155-L157